### DISCUSSION TITLE
Dentist risk factor?

### DISCUSSION THREAD

**Discussion by chelytee** [microCOVID/microCOVID#1492] [General]
I love your website and have used it frequently. I notice that you don't include going to the dentist in your list of activities. Is this because it's just too dangerous?

Thanks!
-CT

**Comment by apiology** [comment]
Hi @chelytee!  I think it's a fair question on what types of things make that list.  I'm not sure I've seen guidelines on what would make the cut to include and what wouldn't, at least in the constraints of the pulldown list we have today.  

If we were to make some, I suppose some inputs could be:
* the total risk of the activities (how high stakes is making a good decision here?) 
* how many changes a person can realistically make in the situation to make it safer (e.g., how MODable is the activity?)
* how common is the activity (how many people will benefit from this taking up real estate on the list vs entering things manually?)
* how hard is this to model independently using our inputs? (how many mis-modelings will we prevent by having helpful presets for this?)
